flexmeasures/ui/templates/views/sensors.html

```diff
@@ -33,6 +33,58 @@
               <div class="alert alert-info" id="dstwarn" style="display:none;"></div>
               <div class="alert alert-info" id="sourcewarn" style="display:none;"></div>
           </div>
+          <div class="container">
+          <div class="row">
+          <div class="col-sm-6">
+                    <h3>Stats</h3>
+                    <table id="statsTable" class="table table-striped">
+                        <tbody id="statsTableBody">
+                        </tbody>
+                    </table>
+                        <script>
+                            fetch('/api/v3_0/sensors/' + {{ sensor.id }} + '/stats')
+                                .then(response => response.json())
+                                .then(data => {
+                                    const tableBody = document.querySelector('#statsTableBody');
+                                    const keys = [
+                                        "min_event_start",
+                                        "max_event_start",
+                                        "min_value",
+                                        "max_value",
+                                        "mean_value",
+                                        "sum_values",
+                                        "count_values"
+                                    ];
+
+                                    for (const sourceKey in data) {
```

Comment:
For each data source, we fill the table? So we are overwriting, only keeping the last source?

I guess the table can become quite large when there is more than one source. We should add a dropdown list above the table to select a source, and on selection only show the rows for that source.
Sorry that this became a bit more complicated.